cmd/mt-kafka-mdm-sniff/main.go

```diff
@@ -20,6 +21,8 @@ import (
 var (
 	confFile = flag.String("config", "/etc/raintank/metrictank.ini", "configuration file path")
 	format   = flag.String("format", "{{.Part}} {{.OrgId}} {{.Id}} {{.Name}} {{.Metric}} {{.Interval}} {{.Value}} {{.Time}} {{.Unit}} {{.Mtype}} {{.Tags}}", "template to render the data with")
+
+	stdoutLock = sync.Mutex{}
```

Comment:
If i see that right `mdm.Start(newInputPrinter(*format))` is never creating more than `1` instance of `inputPrinter`. So couldn't this lock be local to the `inputPrinter` struct? did you just make it package global in case something else wants to print?